backend/src/repositories/ticker_covariance.py

```diff
@@ -1,16 +1,29 @@
 """ Defines the Ticker Covariance repository """
-
+import numpy as np
 from models import TickerCovariance
 
 
 class TickerCovarianceRepository:
+    @staticmethod
+    def get_covariance_matrix(symbols):
+        length = len(symbols)
+        cov_mat = np.ndarray(shape=(length, length))
+        for i in range(length):
+            for j in range(i, length):
+                covariance = TickerCovarianceRepository.get_covariance(
+                    symbols[i], symbols[j]
+                )
+                cov_mat[i][j] = cov_mat[j][i] = covariance[0]
```

Comment:
Could possibly do this for constructing a symmetric matrix
`M + M.T - np.diag(M.diagonal())`
But up to you

Reply:
i'll keep it since this way looks a little neater